examples/app.py

```diff
@@ -17,7 +17,7 @@
    $ pip install -e .[all]
    $ cd examples
    $ mkdir instance
-   $ export FLASK_APP=app.py FLASK_DEBUG=1
+   $ export FLASK_APP=app.py FLASK_ENV=development
```

Comment:
Don't we have to run also `$ flask collect -v`?

Reply:
It is not needed since running create on a project already collects the assets from each bundle, see [here](https://github.com/inveniosoftware/pywebpack/blob/c47ccb5d193ed10be4f5d26cd4409bf3e86eb209/pywebpack/project.py#L205). Reading `pywebpack` docs it is not clear to me that this is happening, should we add more description [there](https://pywebpack.readthedocs.io/en/latest/api.html#pywebpack.project.WebpackBundleProject.create), and maybe also here in Flask-WebpackExt CLI docs? In [this](https://github.com/inveniosoftware/flask-webpackext/pull/7/files#diff-109318e36ab6f53179e11a3aa731c40eR196) brief explanation of what is happening behind the scenes with `buildall` it is kind of explained, do you think it is enough?